ui/src/components/FeaturesListDisplay.tsx

```diff
@@ -1,16 +1,17 @@
-import React from "react";
-import { enabledFeatureStatistics } from "../flags";
+import React, { useContext } from "react";
 import { EuiBasicTable, EuiLoadingSpinner, EuiBadge } from "@elastic/eui";
 import { FeastFeatureColumnType } from "../parsers/feastFeatureViews";
 import useLoadFeatureViewSummaryStatistics from "../queries/useLoadFeatureViewSummaryStatistics";
 import SparklineHistogram from "./SparklineHistogram";
+import FeatureFlagsContext from "../contexts/FeatureFlagsContext";
 
 interface FeaturesListProps {
   featureViewName: string;
   features: FeastFeatureColumnType[];
 }
 
 const FeaturesList = ({ featureViewName, features }: FeaturesListProps) => {
+  const { enabledFeatureStatistics } = useContext(FeatureFlagsContext);
```

Comment:
is this intentional? seems not used so maybe can revert

Reply:
This is intentional. This enables the user to turn on this on as a prop on `<FeastUI>` i.e. 

```jsx
<FeastUI
  feastUIConfigs={{
    featureFlags: {
      enabledFeatureStatistics: true
    },
  }}
/>
```

This already works. I need to document this better hahaha.